src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -18,6 +17,12 @@ import { RouterLink } from "v2/System/Router/RouterLink"
 import { FeatureFeaturedLink_featuredLink } from "v2/__generated__/FeatureFeaturedLink_featuredLink.graphql"
 import { themeGet } from "@styled-system/theme-get"
 
+const Container = styled(Flex)<FlexProps>`
+  &:hover > div {
+    color: ${color("blue100")};
```

Comment:
```suggestion
    color: ${themeGet("colors.blue100")};
```

Reply:
Though, I don't like this, exactly. Shoudn't this be on the `RouterLink`?

Reply:
Yeah makes sense there, I wanted to avoid bc of the use the sibling selector(personal preference), I've changed in this [commit](https://github.com/artsy/force/pull/8501/commits/559e998d999e873f6b55855d42dd608df12b6a04)